delete.js

```diff
@@ -0,0 +1,95 @@
+const { MongoClient } = require('mongodb');
+
+async function main() {
+    /**
+     * Connection URI. Update <username>, <password>, and <your-cluster-url> to reflect your cluster.
+     * See http://bit.ly/NodeDocs_lauren for more details
+     */
+    const uri = "mongodb+srv://<username>:<password>@<your-cluster-url>/test?retryWrites=true&w=majority";
+
+    /**
+     * The Mongo Client you will use to interact with your database
+     * See bit.ly/Node_MongoClient for more details
+     */
+    const client = new MongoClient(uri);
+
+    try {
+        // Connect to the MongoDB cluster
+        await client.connect();
+
+        // Make the appropriate DB calls
+
+        // DELETE ONE
+        // Check if a listing named "Cozy Cottage" exists. Run update.js if you do not have this listing.
+        await printIfListingExists(client, "Cozy Cottage");
+        // Delete the "Cozy Cottage" listing
+        await deleteListingByName(client, "Cozy Cottage");
+        // Check that the listing named "Cozy Cottage" no longer exists
+        await printIfListingExists(client, "Cozy Cottage");
+
+        // DELETE MANY
+        // Check if the listing named "Ribeira Charming Duplex" (last scraped February 16, 2019) exists
+        await printIfListingExists(client, "Ribeira Charming Duplex");
+        // Check if the listing named "Horto flat with small garden" (last scraped February 11, 2019) exists
+        await printIfListingExists(client, "Horto flat with small garden");
+        // Delete the listings that were scraped before February 15, 2019
+        await deleteListingsScrapedBeforeDate(client, new Date("2019-02-15"));
+        // Check that the listing named "Ribeira Charming Duplex" still exists
+        await printIfListingExists(client, "Ribeira Charming Duplex");
+        // Check that the listing named "Horto flat with small garden" no longer exists
+        await printIfListingExists(client, "Horto flat with small garden");
+
+    } finally {
+        // Close the connection to the MongoDB cluster
+        await client.close();
+    }
+}
+
+main().catch(console.error);
+
+/**
+ * Delete an Airbnb listing with the given name.
+ * Note: If more than one listing has the same name, only the first listing the database finds will be deleted.
+ *       It's best to use deleteOne when querying on fields that are guaranteed to be unique.
+ * @param {MongoClient} client A MongoClient that is connected to a cluster with the sample_airbnb database
+ * @param {string} nameOfListing The name of the listing you want to delete
+ */
+async function deleteListingByName(client, nameOfListing) {
+    // See http://bit.ly/Node_deleteOne for the deleteOne() docs
+    result = await client.db("sample_airbnb").collection("listingsAndReviews").deleteOne({ name: nameOfListing });
```

Comment:
`const result =`

Reply:
Ack!  Made the same mistake twice in a row.  Thanks for catching this!